buildbot.mariadb.org/scripts/deb-install.sh

```diff
@@ -1,18 +1,29 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC2154
 
-set -ex
+set -e
+
+# load common functions
+# shellcheck disable=SC1091
+. ./bash_lib.sh
+
+# function to be able to run the script manually (see bash_lib.sh)
+manual_run_switch "$1"
+
+set -x
+
+# print disk usage
 df -kT
 
-# show installed packages (//TEMP see below)
+# show installed packages
 dpkg -l | grep -iE 'maria|mysql|galera' || true
 
 # we want a clean installation here (//TEMP should already be the case, do better here!)
```

Comment:
like starting on a fresh image?

Reply:
Yep this test is useless it's a "should never occur" case, every build is starting a fresh image that is reverted to it's original state when the VM shutdown (see: qemu hook in https://docs.buildbot.net/latest/manual/configuration/workers-libvirt.html#configuring-master-to-use-libvirt-on-remote-server).

I did not removed this because I see just one reason why it could be good to keep it if cloud-init images (or what ever base image we use) comes with `mariadb|mysql|galera `packages already installed, then it may make sense to be able to check this in the run...